src/editor/UI/Tab.re

```diff
@@ -50,10 +59,25 @@ let createElement =
         color(theme.editorForeground),
       ];
 
+    let modifiedStyles =
+      Style.[
+        color(theme.tabModifiedIconColor),
+        marginHorizontal(5),
+        fontSize(fontPixelSize),
+        fontFamily("FontAwesome5FreeRegular.otf"),
```

Comment:
Found out why the circle wasn't working as expected... Turns out both the solid and hollow circle are the same codepoint (`f111`):
- https://fontawesome.com/icons/circle?style=solid
- https://fontawesome.com/icons/circle?style=regular

We just need to change the font here to `FontAwesome5FreeSolid.otf` instead, to pick up the solid version.

Reply:
Cool, glad it worked! 👍